### PR TITLE
feat: configure trading limits & circuit breaker

### LINKED ISSUE #5: Configure trading limits and circuit breaker in the deployment
**Description**

The current deployment script doesn't configure trading limits at all and has very limited circuit breaker configuration.
This PR should extend the configuration with some sane defaults that are meant to:
- Get us closer to a final parameter set
- Help us discover what exactly are the parameters of the system

**Acceptance criteria**

- [x] Refactor the script to use a struct for any variable related to a pool
- [x] Add trading limits and circuit breaker configuration and extend the struct as needed (consult with Nadiem for the exact parameters)
- [x] Document each entry in the struct

### PR DESCRIPTION
**Pull request by bayological**
### Description

This change adds the configuration of the trading limits & circuit breaker to the baklava governance script.

### Other changes

- Added ValueDeltaBreaker to non-upgradeable contracts deployment script.
- Added a struct with helpful comments to hold all config values related to a single pool.
- Added two functions to the simulation swap test to verify configuration worked as expected.

### Tested

- The script was run locally without broadcasting transactions to verify its correctness.
- Simulation was run to verify the configuration steps worked as expected.
- Tests added to verify the configuration of limits and circuit breaker pass. 

### Related issues

- Closes #5 

### Backwards compatibility

N/A

### Documentation

N/A

### CODE REVIEW

File: script/upgrades/MU01-mcmint/governance/MU01-Baklava-CGP.sol [hunk 1]
```diff
@@ -17,26 +17,210 @@ import { Contracts } from "script/utils/Contracts.sol";
...
+    cUSDCeloConfig = PoolConfiguration({
+      asset0: cUSD,
+      asset1: celo,
+      isConstantSum: false,
+      spread: FixidityLib.newFixedFraction(25, 10000), // 0.0025
+      referenceRateResetFrequency: 60 * 5,
+      minimumReports: 5,
+      stablePoolResetSize: 72e23, // 7200000
+      isMedianDeltaBreakerEnabled: true,
+      medianDeltaBreakerThreshold: 3e16, // 0.03
```
Comment (sissnad) [inline script/upgrades/MU01-mcmint/governance/MU01-Baklava-CGP.sol:107]: What is the preferred representation of decimals 
`FixidityLib.newFixedFraction(25, 10000)` versus `25e14`?

Comment (sissnad) [reply script/upgrades/MU01-mcmint/governance/MU01-Baklava-CGP.sol:103]: Similar question regarding time convention: unit free in seconds (`60 * 5 `) versus with time unit (`5 minutes`)

Comment (bayological) [reply script/upgrades/MU01-mcmint/governance/MU01-Baklava-CGP.sol:103]: In both cases, I prefer the latter. For me, it makes them easier to read.

File: script/upgrades/MU01-mcmint/governance/MU01-Baklava-CGP.sol [hunk 2]
```diff
@@ -15,28 +15,218 @@ import { IRegistry } from "mento-core/contracts/common/interfaces/IRegistry.sol"
...
+      medianDeltaBreakerThreshold: FixidityLib.newFixedFraction(3, 100), // 0.03
+      medianDeltaBreakerCooldown: 30 minutes,
+      isValueDeltaBreakerEnabled: false,
+      valueDeltaBreakerThreshold: FixidityLib.wrap(0),
+      valueDeltaBreakerReferenceValue: 0,
+      valueDeltaBreakerCooldown: 0,
+      referenceRateFeedID: cBRL,
+      asset0_timeStep0: 5 minutes,
+      asset0_timeStep1: 1 days,
+      asset0_limit0: int48(1_000_000),
```
Comment (sissnad) [inline script/upgrades/MU01-mcmint/governance/MU01-Baklava-CGP.sol:184]: Sorry, for picking on conventions again...Is there a reason why the cBRL limits are typed?

**Review by sissnad — APPROVED** [review]
looks good to me

File: script/upgrades/MU01-mcmint/governance/MU01-Baklava-CGP.sol (hunk 2 above)
Comment (tobikuhlmann) [reply script/upgrades/MU01-mcmint/governance/MU01-Baklava-CGP.sol]: I did this to solve some implicit type conversion bug. Maybe there's a cleaner solution

**Pull request merged**